tests/Fixture/BakeTemplateRolesFixture.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Created by javier
+ * Date: 31/01/16
+ * Time: 10:59
+ */
```

Comment:
Oh, I missed that one. It's the default header added by the IDE, besides that I've replaced on other new files, I didn't know what to put in @since, so if you can take care of that too, go ahead.